spec/element_base_spec.rb

```diff
@@ -66,6 +66,12 @@
     end
   end
 
+  describe '#countable?' do
+    it 'returns false if not countable (default true)' do
```

Comment:
spec description is kind of confusing.  `it 'defaults to true'` would be clearer